readme.md

```diff
@@ -4,7 +4,7 @@
 
 ## Synopsis
 
-This project is a serverless implementation to provide CAP XML services, through the use of AWS Lambda.
+This project is implementation to provide CAP XML services, through the use of AWS Lambda.
```

Comment:
I suggest "This project provides CAP XML services through the use of AWS Lambda."